gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -60,12 +60,18 @@ <h2 class="title">Affected packages</h2>
     <spicy-sections
       class="vulnerability-packages{% if vulnerability.affected|should_collapse %} force-collapse{% endif %}">
       {% for affected in vulnerability.affected -%}
+      {% if 'package' in affected %}
```

Comment:
nit: Can this be indented? Or will the formatter always unindent it.

Reply:
> nit: Can this be indented? Or will the formatter always unindent it.

The formatter very helpfully unindented it all for me after I'd made it way more readable at the time 🤬